csr/Makefile

```diff
@@ -0,0 +1,21 @@
+VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+[^0-9]\([0-9]\+\.[0-9]\+\.[0-9]\+[sb]\?\?\)\([^0-9].*\|$$\)/\1/')
```

Comment:
Please include sample file names that the regex matches. It's much easier afterwards, when editing the regex, if we have some examples that we can make sure we still match..